runtime/shared_common/CacheMap.cpp

```diff
@@ -4303,18 +4300,44 @@ SH_CacheMap::releasePrivateSharedData(J9VMThread* currentThread, const J9SharedD
 }
 
 /* THREADING: Can be called at any time by any thread. Should not try to get any locks as it may be being
- * called as a result of a deadlock. The only locks obtained are by the managers when querying their hashtables. */
+ * called as a result of a deadlock. The only locks obtained are by the managers when querying their hashtables.
+ * 
+ * Get the J9SharedClassJavacoreDataDescriptor from the cache of all layers.
```

Comment:
This needs to be worded better. Like, Get the J9SharedClassJavacoreDataDescriptor for all layers of the cache.

Reply:
Fixed